src/pages/Settings.js

```diff
@@ -20,6 +20,29 @@ export default function Settings({ api, setUser }) {
   const history = useHistory();
   const user = useContext(UserContext);
   const [languages, setLanguages] = useState();
+  
+  
+  // TODO: Refactor using Zeeguu project logic
+  const sysLanguages = [
+    {
+      name: "Danish",
+      code: "da",
+    },
+    {
+      name: "English",
+      code: "en",
+    },
+  ];
+
+  const [sysLanguage, setSysLanguage] = useState();
```

Comment:
maybe `uiLanguage`? system is everything... :)

Reply:
Sure :)